protocol.txt

```diff
@@ -35,5 +35,9 @@ ACCEPTLOAN <Loan Number> <Account>
 e.g. ACCEPTLOAN 1 11111111
 Returns: Success. Loan number 1 accepted by account 11111111.
```

Comment:
Ah great, that you thought of this change. While we're at it, could you also update the HELP command?

Reply:
nicely done

Reply:
Thanks @Jabul99